index.html

```diff
@@ -8,7 +8,180 @@
+          <div>
+            <img src="./images/forum.png" alt="Forum" />
+            <h3>Forum</h3>
+            <p>
+              Take the time to speak with professionals about a certain subject
+              and engage in discussions with them.
+            </p>
+          </div>
+          <div>
+            <img src="./images/workshop.png" alt="Workshop" />
+            <h3>Workshop</h3>
+            <p>
+              Demonstrate the utility of AI in various industries and engage in
+              detailed discussions.
+            </p>
+          </div>
+          <div>
+            <img src="./images/ignite.png" alt="AI ignite" />
+            <h3>AI connect</h3>
+            <p>
+              Get opportunities to network with and share experiences with
+              people from all over the world.
+            </p>
+          </div>
+          <button class="joinbutton" hidden>Join African AI summit 2023</button>
+        </div>
+        <a href="#" class="desktopprogram">SEE THE WHOLE PROGRAM</a>
+      </section>
+      <!--Featured speaker -->
+      <section class="speaker" id="speakers">
+        <!-- 1 -->
+        <h2 class="speakertitle">Featured Speakers</h2>
+        <hr class="outline" />
+        <div class="speakerposition">
+          <!-- all cards generated and populate data using javascript  -->
+          <div class="speakercard">
+            <img class="speakerpicture" />
+            <div>
+              <h3 class="speakername"></h3>
+              <h4 class="speakertitledetail"></h4>
+              <hr class="speakeroutline" />
+              <p class="speakerexperience"></p>
+            </div>
+          </div>
+          <div>
+            <button class="speakerbutton" id="morebutton">
+              MORE <i class="fa-solid fa-angle-down"></i>
+            </button>
+            <button class="speakerbutton" id="lessbutton" hidden>
+              LESS <i class="fa-solid fa-angle-up"></i>
+            </button>
+          </div>
+        </div>
+      </section>
+      <!-- sponsor -->
+      <section class="sponsor homesponsor" id="partners">
+        <h2>Partner</h2>
+        <hr class="partneroutline" />
+        <div class="sponsorimg">
+          <img
+            src="./images/Addis_Ababa_University_logo.png"
+            alt="addis ababa university"
+          />
+          <img
+            src="./images/EthiopianAI.png"
+            alt="Ethiopian Artificial Intelligence Institute"
+          />
+          <img src="./images/Presight_AI.png" alt="Presight AI" />
+          <img src="./images/Hawk university.png" alt="Hawk university" />
+          <img src="./images/IcogLab.png" alt="IcogLab" />
+        </div>
+      </section>
+    </main>
+    <footer class="homefooter">
+      <img
+        class="imglogo"
+        src="./images/ailogo.png"
+        alt="afirca AI summit 2023 logo"
+      />
+      <p class="footerdetail">2023 Africa AI Conference.All rights reserved.</p>
+    </footer>
```

Comment:
- Good job done so far, but there are some little differences between your work and the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

Your Project:
- You have aligned the footer wrongly, place make sure the logo and text are aligned side by side 👍 
<img width="960" alt="xtc" src="https://user-images.githubusercontent.com/73607512/210746405-0d8cd467-af29-420b-a183-29a89fd96eb7.PNG">

Design Template:
<img width="628" alt="htewt" src="https://user-images.githubusercontent.com/73607512/210746887-aec86723-4298-4be1-8750-dd809c077e5f.PNG">

Reply:
@mckent05  Many thanks for the feedback; the footer elements has been fixed.